app/services/accept_offer.rb

```diff
@@ -0,0 +1,29 @@
+class AcceptOffer
```

Comment:
Should we start using the namespaces for the services as the number of services is growing?

Reply:
Added here https://trello.com/c/mDVMOZZx/367-perhaps-use-a-namespace-for-the-services

Reply:
👍